atest/robot/running/skip.robot

```diff
@@ -118,6 +118,12 @@ Skipped with --SkipOnFailure when Failure in Test Teardown
 Skipped with --SkipOnFailure when Set Tags Used in Teardown
     Check Test Case    Skipped with --SkipOnFailure when Set Tags Used in Teardown
 
+Skipped when test is tagged with robot:skip
+    Check Test Case    ${TEST NAME}    status=SKIP
+
+Skipped although test fails since test is tagged with robot:skip-on-failure
+    Check Test Case    ${TEST NAME}    SKIP
+
```

Comment:
- No need to pass status to `Check Test Case` because tests in the `atest/tesdata` have it in their documentation.
- First test could be right after `Skipped with --skip` test because they are highly related. The latter could be here after various `--skiponfailure` tests. Test on the data side should be organized the same way.